src/OembedServiceProvider.php

```diff
@@ -2,26 +2,32 @@
 
 namespace Awcodes\FilamentOembed;
 
-use Filament\PluginServiceProvider;
+use Filament\Support\Assets\Css;
+use Filament\Support\Facades\FilamentAsset;
 use Spatie\LaravelPackageTools\Package;
+use Spatie\LaravelPackageTools\PackageServiceProvider;
 
-class OembedServiceProvider extends PluginServiceProvider
+class OembedServiceProvider extends PackageServiceProvider
 {
     public static string $name = 'filament-oembed';
 
-    protected array $styles = [
-        'plugin-filament-oembed' => __DIR__.'/../resources/dist/filament-oembed.css',
-    ];
-
-    protected array $scripts = [
-        'plugin-filament-oembed' => __DIR__.'/../resources/dist/filament-oembed.js',
-    ];
-
     public function configurePackage(Package $package): void
     {
         $package->name(static::$name)
             ->hasViews()
             ->hasAssets()
             ->hasTranslations();
     }
+
+    public function packageBooted()
+    {
+        parent::packageBooted(); // TODO: Change the autogenerated stub
+
+        FilamentAsset::register([
+            Css::make(
+                'plugin-filament-oembed',
+                __DIR__.'/../resources/dist/filament-oembed.css'
+            ),
+        ], 'awcodes/filament-oembed');
+    }
```

Comment:
This whole method can be removed, since Filament v3 recommends using plugins with a custom theme now. I'll take care of the Readme if you can handle this. 

Thanks.